src/css/custom.css

```diff
@@ -112,6 +112,12 @@ a.pills__item:not(.pills__item--active) {
   fill: var(--ifm-font-color-base);
 }
 
+/* To toggle the visibility of an image using the path fragment */
+[data-theme='light'] img[src$='#gh-dark-mode-only'],
+[data-theme='dark'] img[src$='#gh-light-mode-only'] {
```

Comment:
I like this solution, but what does "gh" stand for in the hash? Maybe `hidden--light-mode` would be a better name to align with the desktop/mobile utility classes. 🙂

Reply:
Good suggestion 👍. Actually, `gh` means `Github`. https://docusaurus.io/docs/next/markdown-features/assets#github-style-themed-images